### DISCUSSION TITLE
What's the vision of Pekko?

### DISCUSSION THREAD

**Discussion by sigurd-cp** [apache/pekko#1519] [General]
I'm thinking about vision of Pekko. What should be reached in 5-10 years?
Besides the important task of maintaining such a big project: 
Are there ideas about fundamental changes (thinking about Java Virtual Threads,...)?
Is there a long-term roadmap (planned)?

**Comment by He-Pin** [comment]
For the virtual threads, We at taobao, just migrate many server apps to Java 21, and I have to say, you need to wait maybe JDK25 to let it works more smothly in production, you know ,dead locks:(

I did a quic poc and mostly done I think, there is a PR there, I will pick it up later
